Signal/src/AppDelegate.m

```diff
@@ -239,7 +239,11 @@ - (void)applicationDidBecomeActive:(UIApplication *)application {
                                                [TSSocketManager becomeActiveFromForeground];
                                                [[Environment getCurrent].contactsManager verifyABPermission];
                                            }];
-
+    
+    if ([TSAccountManager isRegistered]) {
+        [[PushManager sharedManager] applicationDidBecomeActive];
```

Comment:
If there is a blocking migration, e.g. if we add a new database view, it can take a long time for `[TSAccountManager isRegistered]` to return, so in AppDelegate I've gotten into the practice of putting things that read from the DB into an async block like #235.